stake-pool/program/Cargo.toml

```diff
@@ -22,7 +23,10 @@ spl-token = { path = "../../token/program", features = [ "no-entrypoint" ] }
 thiserror = "1.0"
 
 [dev-dependencies]
+bincode = "1.3.1"
+solana-program-test = "1.4.5"
```

Comment:
Bump this to v1.4.7 to match solana-sdk (happened today :) )

Reply:
Will do, thanks!